src/fragment-components/a-fragment.tsx

```diff
@@ -13,14 +13,14 @@ import { GlobalStateContext } from '../context';
 import { classNameFromFragment, tagNameFromFragment } from '../utils/fragment-tools';
 import { LinkButton } from '../components';
 
-export const AFragmentStyleUI = ({selectedComponent, setComponent}: any) => {
+export const AFragmentStyleUI = ({ selectedComponent, setComponent }: any) => {
 	return <>
 		<LinkButton
-		kind='secondary'
-		size='sm'
-		renderIcon={Edit32}
-		className={css`margin-bottom: 1rem`}
-		to={selectedComponent.id}>
+			kind='secondary'
```

Comment:
No indentation.

Reply:
Should I also indent Checkbox? I'm guessing we only need to align the props with the Component if it has children?